docs/standard-library/openapi3/openapi.md

```diff
@@ -267,14 +267,13 @@ that a union should be emitted as a `oneOf` rather than `anyOf`.
 
 When working with the `@encode` decorator the rule is as follow. Given the 3 values `encoding`, `encodeAs` and `realType` where `@encode(encoding, encodeAs) _: realType`:
 
-1. Is one of those special cases:
-
-   - `unixTimestamp` encoding will produce `type: integer, format: unixtime`
-   - encoding a `utcDateTime` or `offsetDateTime` will produce this format `date-time-<encoding>` (e.g. `date-time-rfc7231` for `rfc7231` encoding)
-
-2. When `encodeAs` is specified, it will be used to generate the corresponding schema (e.g. encoding as `int32` will produce an `type: integer, format: int32` )
-
-3. Otherwise use the `encoding` as the format
+1. If `realType` is `utcDateTime` or `offsetDateTime`:
+   - `encoding` of `rfc3339` will produce `type: string, format: date-time`
+   - `encoding` of `rfc7231` will produce `type: string, format: date-time-rfc7231`
```

Comment:
I thought we said we'll make this do `http-date` instead

Reply:
We could do that. But we'd keep the special case in autorest, right?  (I'm working on that PR now)

Reply:
yep keep for autorest but make openapi3 follow the conventions as much as possible